### ISSUE TITLE
ISSUE - ExtractArchive/Send2Lib - Disabled "Move to trash" not working

### ISSUE THREAD

**Issue by Ryan747800B** [pyload/pyload#2623]
`Move to trash instead delete` option doesn't stay disabled after a reboot, even if deactivated from `plugin.conf`. How can I force the `off` option?

```
ExtractArchive - "ExtractArchive":
    bool deltotrash : "Move to trash instead delete" = False
    bool repair : "Repair broken archives (RAR required)" = False
    bool activated : "Activated" = True
    bool overwrite : "Overwrite files" = True
    bool waitall : "Run after all downloads was processed" = False
    bool recursive : "Extract archives in archives" = True
    bool keepbroken : "Try to extract broken archives" = False
    folder destination : "Extract files to folder" = 
    int priority : "Process priority" = 0
    file passwordfile : "Password file" = unrar_passwords.txt
    str excludefiles : "Don't extract the following files" = *.nfo,*.DS_Store,index.dat,thumb.db
    bool usepasswordfile : "Use password file" = True
    bool subfolder : "Create subfolder for each package" = False
    str extensions : "Extract archives ending with extension" = 7z,bz2,bzip2,gz,gzip,lha,lzh,lzma,rar,tar,taz,tbz,tbz2,tgz,xar,xz,z,zip
    bool fullpath : "Extract with full paths" = True
    bool delete : "Delete archive after extraction" = True
```

**Comment by GammaC0de** [comment]
There is a strange bug in v0.4.9 that sometimes settings are not persistent.
Just ignore this, it doesn't worth the hassle, its just a warning.
Hopefully it will fixed in the next version (v0.5)

**Comment by Ryan747800B** [comment]
:+1:  Thank you. Waiting for a 0.5 beta version!
